doc/conf.py

```diff
@@ -241,3 +241,5 @@
 # Prevent label issues due to colliding section names
 # through including multiple documents
 autosectionlabel_prefix_document = True
+
+linkcheck_ignore = ["https://github.com/PyCQA/pylint/blob/main/pylint/extensions/.*"]
```

Comment:
Could we check the extension on ``main`` (where the extension should be published already) but not on feature branch by creating a CI job specific for this test ?

Reply:
Yeah, but then we're likely to miss failures on the step right? I don't think these links will ever break, so it should be fine to just ignore these very specific links in normal CI runs?

Reply:
Well, 2 weeks ago we did not check ANY links, I think we'll be fine 😄

Reply:
All the choices are somewhat flawed :-)